apps/website/src/pages/show-and-tell/give-an-hour.tsx

```diff
@@ -78,7 +78,7 @@ const GiveAnHourPage: NextPage = () => (
   <>
     <Meta
       title="Give an Hour | Show and Tell"
-      description="Join the Alveus community and WWF to Give an Hour for Earth! Discover what actions you can take to help the environment and wildlife."
+      description="Join the Alveus community and give an hour for earth! Discover what actions you can take to help the environment and wildlife."
```

Comment:
Not sure, but maybe "Earth" (with capital e) is better here and at the other changed places? We talk here about Earth about the named place and not "dirt". (Please let me know if this comment is ok... I am used to do pull requests / code reviews at work and never done it in an open source setting)

Reply:
Of course its okay and appreciated! :)

Reply:
Thanks, looks nice now. Do I have to close this conversation or accept that somehow? (Gitlab user here with :D)